export/block-content.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="26" height="25" viewBox="0 0 26 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
For consistency, the `viewBox` of this SVG should be `0 0 25 25` – similarly, width should be `25`

Reply:
(You may also need to revisit paths after changing this!)